Tests/interop/net/type/test_reachtype.py

```diff
@@ -189,65 +191,67 @@ def test_type_from_reflection_emit():
         import EmittedNS
         EmittedNS.EmittedType()
     
-def test_type_forward1():
-    add_clr_assemblies("typeforwarder1")
-    from NSwForwardee1 import Foo, Bar        #!!!
-    AreEqual(Foo.A, 120)
-    AreEqual(Bar.A, -120)
-    
-    import NSwForwardee1
-    AreEqual(NSwForwardee1.Foo.A, 120)
-    AreEqual(NSwForwardee1.Bar.A, -120)
-
-@skip("multiple_execute", "posix")
```

Comment:
In other places you kept the multiple_execute as a TODO however it seems you zapped it here.